README.md

```diff
@@ -27,12 +27,12 @@ The Selenium tests are run on different platforms like on-prem, docker and Brows
 
   Maven:
     ```sh
-    mvn install
```

Comment:
Why skipping tests? if they are failing they should be fixed.

Reply:
intention is not to skip the tests to have build and test commands separated out.
the test commands are already called out as we progress through the readme. 
the install command with tests will give users a bad experience since they haven't followed the "Prerequisites" section for the default profile of the readme yet

which is here: [Prerequisites](https://github.com/browserstack/browserstack-examples-cucumber-testng#prerequisites)